src/data/polynomial/basic.lean

```diff
@@ -129,7 +143,7 @@ lemma coeff_sub (p q : polynomial R) (n : ℕ) : coeff (p - q) n = coeff p n - c
 
 end ring
 
-instance [comm_semiring R] : comm_semiring (polynomial R) := add_monoid_algebra.comm_semiring
+-- instance [comm_semiring R] : comm_semiring (polynomial R) := add_monoid_algebra.comm_semiring
```

Comment:
Is there a reason to comment this out?

Reply:
I moved this line to line 123.

Reply:
Ah, oops. OK, please delete the line completely:
```suggestion
```